node_address.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/hashgraph/hedera-protobufs-go/services"
 )
 
-type _NodeAddress struct {
+type NodeAddress struct {
```

Comment:
I'm very conflicted about this change as I hacked `NodeAddress` together real quick and didn't consider it for a public API meaning we might be able to design a better `NodeAddress` if we want it to be publicly available. For starters we should make all the fields public.